templates/components/common/navigation-menu.html

```diff
@@ -2,15 +2,26 @@
     <div class="navPages-quickSearch">
         {{> components/common/quick-search}}
     </div>
-    <ul class="navPages-list">
+    <ul class="navPages-list{{#if theme_settings.menu_display_depth_max}} navPages-list-depth-max{{/if}}">
```

Comment:
It would be more readable if we have something like:

```
{{#if theme_settings.navigation_design '==' 'dropdown'}}
  {{> {{> components/common/navigation-dropdown}}
{{/if}}
{{#if theme_settings.navigation_design '==' 'columns'}}
  {{> {{> components/common/navigation-columns}}
{{/if}}
```

This way it would be easier to add a third design in the future